README.md

```diff
@@ -769,12 +769,12 @@ in the JetStream examples for a detailed and runnable example.
 **Iterate:**
 
 ```java
-        Iterator<Message> iter = sub.iterate(100, Duration.ofSeconds(1));
-        while (iter.hasNext()) {
-            Message m = iter.next();
-            // process message
-            m.ack();
-        }
+Iterator<Message> iter = sub.iterate(100, Duration.ofSeconds(1));
+while (iter.hasNext()) {
+    Message m = iter.next();
+    // process message
+    m.ack();
+}
```

Comment:
The way this is indented is intentional and matches the other indentation in the page. You can see that in the Fetch example just prior to this example. Please remove this change and I will accept the PR.

Reply:
Oh, I see.
Fixed